src/regex/regex.ts

```diff
@@ -0,0 +1,8 @@
+export const regexPatterns = {
+  email: /^[\w-]+(\.[\w-]+)*@([\w-]+\.)+[a-zA-Z]{2,7}$/,
+  cpf: /^\d{3}\.\d{3}\.\d{3}-\d{2}$/,
+  cnpj: /^\d{2}\.\d{3}\.\d{3}\/\d{4}-\d{2}$/,
+  specialCharacter: /[^A-Za-z0-9]/,
+  uppercaseLetter: /[A-Z]/,
+  numericDigit: /\d/,
+};

```

Comment:
usou isso pra que?

Reply:
Eu tinha usado ele, mais aí falei contigo na call sobre como é a validação de login. e resolvi deixar pra usar quando for fazer a validação de cadastro

então deixei ele aí mesmo, mais pra frente eu usei ele